### ISSUE TITLE
How to Train on other loss functions

### ISSUE THREAD

**Issue by hamed-tgh** [feixiang7701/ConvCRF#4]
hi, thanks for implementing Convcrf in tensorflow.
I want to use your model in an end to end network and try other probabilistic loss. would you tell me how can i train convcrf in end to end network?

best regards
